packages/upload/Upload.js

```diff
@@ -146,6 +146,7 @@ class Upload extends Component {
       maxSize,
       children,
       showFileDrop,
+      isDisabled,
```

Comment:
Let's add this prop to the README

https://github.com/Availity/availity-react/tree/master/packages/upload#props